modules.d/30convertfs/convertfs.sh

```diff
@@ -50,7 +50,7 @@ if [ ! -L "$ROOT"/var/lock -a -e "$ROOT"/var/lock ]; then
     ln -sfn ../run/lock "$ROOT"/var/lock
 fi
 
-[ -n $SUBVOLIDVAR ] && umount $ROOT/var
+[[ -n $SUBVOLIDVAR ]] && umount $ROOT/var
 [ -w $ROOT ] && mount -o remount,ro $ROOT
```

Comment:
Could you also fix this like in the shellcheck commit in pr 355?

Reply:
Do you mean adding also the whole shellcheck commit to all code streams or just this line? I haven't added the whole commit to old code streams on purpose because I think is more "a code improvement" than a bug fix, and it would be submitted via maintenance update, but I can submit it also if you want.

If your concern is the 2 lines with `[ -w $ROOT ]`, the script exits at the beginning if `$ROOT` is not a directory.

https://github.com/openSUSE/dracut/blob/20b345b4e1bcaf1ea71cc67263b13cb4d8e85577/modules.d/30convertfs/convertfs.sh#L5-L8

Reply:
Yeah, just wanted to avoid possible errors due to missing double square brackets.
But if it's only cosmetic I'm fine without.